includes/class-wc-payments-account.php

```diff
@@ -1230,7 +1231,16 @@ public function update_stripe_account( $stripe_account_settings ) {
 				Logger::info( 'Skip updating account settings. Nothing is changed.' );
 				return;
 			}
-			$updated_account = $this->payments_api_client->update_account( $stripe_account_settings );
+
+			$request = Update_Account::create();
+			// Dynamically set the request parameters.
+			foreach ( $stripe_account_settings as $param => $value ) {
```

Comment:
Instead of looping, you can create a static function and fill the request from that function. In this example, the code can trigger an exception if one of the param doesn't have a setter function. You can take a look at the Paginated request class and see how it is done there.

Reply:
Another great feedback with the idea of creating a static function! Addressed at 9be5c979d8f967cc7c0308637035cda37d1531d3